packages/sdk/client-services/src/packlets/spaces/data-space-manager.ts

```diff
@@ -211,6 +200,53 @@ export class DataSpaceManager {
     return space;
   }
 
+  async isDefaultSpace(space: DataSpace): Promise<boolean> {
+    const rootDoc = await this._getSpaceRootDocument(space);
+    for (const object of Object.values(rootDoc.docSync()?.objects ?? {}) as ObjectStructure[]) {
```

Comment:
There's a function for that: `findPropertiesObject`

Reply:
nice